src/static/css/cfpb-chart-builder.less

```diff
@@ -1,4 +1,4 @@
-@import 'https://code.highcharts.com/css/highcharts.css';
+@import 'highcharts.less';
```

Comment:
groovy i'll give it a go. makes sense to me instead of manually duplicating this and maintaining it ourselves!